src/components/Helper.vue

```diff
@@ -78,7 +78,7 @@ Object.keys(explanations).forEach((urlKey) => {
 
 const getHelpInformations = (currentPath, translatedExplanations) => {
   let currentPathKey = currentPath.slice(1);
-  if (!(currentPathKey in translatedExplanations)) {
+  if (Object.keys(translatedExplanations).findIndex(currentPathKey) === -1) {
```

Comment:
```suggestion
  if (!Object.keys(translatedExplanations).includes(currentPathKey)) {
```
Why don't we use includes?

Reply:
Because I forget the existence of this method ;)